tests/model_fields/test_integerfield.py

```diff
@@ -90,6 +90,54 @@ def test_backend_range_validation(self):
             instance.value = max_value
             instance.full_clean()
 
+    def test_redundant_backend_range_validators(self):
+        """
+        If there are stricter validators than ones from backend
+        then backend validators are not added
+        (#26786)
+        """
+        min_backend_value, max_backend_value = self.backend_range
+        ranged_field = self.model._meta.get_field('ranged_value')
```

Comment:
You could create a new field instead of requiring an extra column with an arbitrary default on existing models.

e.g.

`field = self.model._meta.get_field('value').__class__(validators=[MinValueValidator(1), MaxValueValidator(3)])`

And remove `ranged_field` from all the models.